kotlin/src/test/kotlin/org/jdbi/v3/core/kotlin/KotlinMapperTest.kt

```diff
@@ -554,4 +555,30 @@ class KotlinMapperTest {
             .first())
             .isEqualTo(ClassWithNullableNestedProperty(1).also { it.nested = NestedDataClass("foo", "bar") })
     }
+
+    @PropagateNull("name")
+    class ClassWithPropagateNull(val id: Int) {
+        var name:String? = null
+    }
+
+    @Test
+    fun classPropagateNull() {
+        assertThat(handle.select("select 1 as id, null as name")
+                .mapTo<ClassWithPropagateNull>()
+                .one())
+            .isNull()
+    }
+
+    class MemberWithPropagateNull(val id: Int) {
+        @PropagateNull
+        var name:String? = null
+    }
+
+    @Test
+    fun memberPropagateNull() {
+        assertThat(handle.select("select 1 as id, null as name")
+                .mapTo<MemberWithPropagateNull>()
+                .one())
+            .isNull()
+    }
```

Comment:
Can we also get a test for nullable constructor parameters with `@PropagateNull`? Or is that not a valid use case?